src/lib.rs

```diff
@@ -345,11 +347,15 @@ pub mod pallet {
 
 			let current_initialized_rewards = InitializedRewardAmount::<T>::get();
 
-			// This ensures that the rewards match the pot
-			ensure!(
```

Comment:
Shouldn't we keep this ensure with a more relaxed check like:
```
// I suppose there can't be more than 1 unit (10^-18) dust per reward so
current_initialized_rewards > Self::pot() - TotalContributors::<T>::get().len()
```

Reply:
makes total sense